metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: DuplicateStorageAutoScalerCRsFound
+      annotations:
+        description: |-
+          Duplicate StorageAutoScaler CRs found in the namespace {{ $labels.namespace }} trying to scale device class {{ $labels.device_class }} under storage cluster {{ $labels.storagecluster }}.
+        message: Duplicate StorageAutoScaler CRs found.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/DuplicateStorageAutoScalerCRsFound.md
```

Comment:
Run books URL yet to be created.
@parth-gr , @ShravaniVangur , please take a look

Reply:
@ShravaniVangur can you work upon the runbooks